medusa/purge.py

```diff
@@ -62,6 +70,58 @@ def main(config, max_backup_age=0, max_backup_count=0):
         sys.exit(1)
 
 
+def find_backups_to_purge(
+        complete_cluster_backup_names, node_backups, prefer_incomplete, max_backup_age, max_backup_count
+):
+    backups_to_purge = set()
+
+    # we purge old backups always
+    backups_to_purge |= set(backups_to_purge_by_age(node_backups, max_backup_age))
```

Comment:
**Question:** So we're considering that incomplete backups shouldn't be affected by the new purge priorities that we're setting? How is that purge rule different than the purge by count rule?